docs/en/faq/common-issues-in-deployment-and-development-phase.md

```diff
@@ -253,3 +253,39 @@ location /apollo/ {
     proxy_pass http://127.0.0.1:8070/;
 }
 ```
+
+### 17. How to configure https after Portal is mounted to nginx/slb?
+
+Modify application-github.properties under the config directory in the apollo-portal installation package and add the following configuration:
+
+```properties
+server.forward-headers-strategy=framework
+```
+
+It can also be configured through environment variables:
+
+```
+SERVER_FORWARD_HEADERS_STRATEGY=framework
+```
+
+Then configure https on nginx/slb, taking nginx as an example:
+```
```

Comment:
the nginx conf example missing the redirect section
```
server {
    listen 80 default_server;

    location / {
        # redirect all requests on port 80 to https
        return 301 https://$http_host$request_uri;
    }
}
```

Reply:
How about provide two examples ?
One without redirect section and the other with redirect section.

Reply:
This is a good idea, I've updated the documentation with both examples.